diff -up chrpath-0.13/main.c\~ chrpath-0.13/main.c
--- chrpath-0.13/main.c~	2013-04-04 18:01:29.519912735 +0200
+++ chrpath-0.13/main.c	2013-04-08 10:34:19.332766127 +0200
@@ -52,6 +52,7 @@ usage(char *progname)
   printf("   -r <path>|--replace <path>  Replace current rpath/runpath setting\n");
   printf("                               with the path given\n");
   printf("   -l|--list                   List the current rpath/runpath (default)\n");
+  printf("   -k|--keepgoing              Do not fail on first error\n");
   printf("   -h|--help                   Show this usage information.\n");
 #ifndef HAVE_GETOPT_LONG
   printf("\n *** The long options are not available on this platform");

Diff finished.  Mon Apr  8 10:34:27 2013
